var/spack/repos/builtin/packages/r/package.py

```diff
@@ -62,6 +62,8 @@ class R(AutotoolsPackage):
             description='Links to externally installed BLAS/LAPACK')
     variant('X', default=False,
             description='Enable X11 support (call configure --with-x)')
+    variant('memory-profiling', default=False,
```

Comment:
Can you use an underscore instead of a dash? The parser won't have any problems, but `-memory-profiling` is ambiguous to humans whether it's one or two variants.